patches/chrome-browser-resources-side_panel-bookmarks-power_bookmarks_list.ts.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
+index 897919f0015c30e247d10872ae031dc059f413f0..e1e3f74ead610b18919dcf75fdc4ab400cf38a34 100644
+--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
++++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
+@@ -171,7 +171,8 @@ export class PowerBookmarksListElement extends PolymerElement {
+                sortOrder: SortOrder.kReverseAlphabetical,
+                label: loadTimeData.getString('sortReverseAlphabetically'),
+                lowerLabel: loadTimeData.getString('sortReverseAlphabetically'),
+-             }],
++             },
++             { sortOrder: SortOrder.kCustom, label: loadTimeData.getString('sortCustom'), lowerLabel: loadTimeData.getString('sortCustomLower'), }],
```

Comment:
Hey, I'm not 100% sure, but I think you might be able to override this without patching by doing something like this:

```ts
const originalProperties = PowerBookmarksListElement.properties

delete PowerBookmarksListElement.properties

Object.defineProperty(PowerBookmarksListElement, "properties", {
    get() {
        [...originalProperties, { sortOrder: SortOrder.kCustom, label: loadTimeData.getString('sortCustom'), lowerLabel: loadTimeData.getString('sortCustomLower'), }]
    }
}
```

you might have to play around with it a little bit but I think it should work.

Reply:
I'm trying to remove this patch with https://github.com/brave/brave-core/pull/21361/commits/06e2549c5cc7f789ac427ffc10169eaef0b95894 but still no luck..

Reply:
I just had a play around and yeah, it seems to struggle patching `properties` :cry: